cross/sslh/Makefile

```diff
@@ -1,42 +1,26 @@
 PKG_NAME = sslh
-PKG_VERS = 1.13
-SRC_VERSION = 1.13b
+PKG_VERS = v1.16
+SRC_VERSION = v1.16
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(SRC_VERSION).$(PKG_EXT)
 PKG_DIST_SITE = http://www.rutschle.net/tech
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 SRC_DIR= $(WORK_DIR)/$(PKG_NAME)-$(PKG_VERS)/
-VERSION="v1.13b"
+VERSION="v1.16"
```

Comment:
Seems this is no longer needed, right? Otherwise, using `${PKG_VERS}` would be just as good.